Sources/NIOExtras/DebugInboundEventsHandler.swift

```diff
@@ -0,0 +1,95 @@
+import NIO
+
+public class DebugInboundEventsHandler: ChannelInboundHandler {
+    
+    public typealias InboundIn = ByteBuffer
```

Comment:
This is incorrect. The correct two typealiases are `InboundIn = Any` and `InboundOut = Any`.